code/game/objects/items/circuitboards/machine_circuitboards.dm

```diff
@@ -373,16 +372,15 @@
 	needs_anchored = FALSE //wew lad
 	var/secure = FALSE
 
-/obj/item/circuitboard/machine/holopad/attackby(obj/item/P, mob/user, params)
-	if(P.tool_behaviour == TOOL_MULTITOOL)
-		if(secure)
-			build_path = /obj/machinery/holopad
-			secure = FALSE
-		else
-			build_path = /obj/machinery/holopad/secure
-			secure = TRUE
-		to_chat(user, span_notice("You [secure? "en" : "dis"]able the security on the [src]"))
-	. = ..()
+/obj/item/circuitboard/machine/holopad/multitool_act(mob/living/user, obj/item/tool)
+	. = TRUE
```

Comment:
Change to just `return TRUE` at the end, `.` can be a bit of a hack but this isn't serving a purpose.

Reply:
fair enough, i just wasn't sure how `$TOOL_act` worked and thats how i saw every use of it being done